src/algebra/ordered_group.lean

```diff
@@ -623,6 +623,8 @@ instance : decidable_linear_ordered_cancel_comm_monoid α :=
   add_right_cancel := λ x y z, add_right_cancel,
   ..s }
 
+lemma eq_of_abs_non_pos {a b : α} (h : abs (a - b) ≤ 0) : a = b :=
+eq_of_abs_sub_eq_zero (le_antisymm _ _ h (abs_nonneg (a - b)))
```

Comment:
new line, and the name `eq_of_abs_sub_non_pos` is more descriptive I think

Reply:
I'm not sure where you want a new line. After Line 627?

Reply:
Ok, I also realized the name has an extra underscore in `nonpos`. It should be fine now